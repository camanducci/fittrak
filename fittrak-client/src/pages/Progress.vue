<template>
  <v-container>
    <v-layout 
      row 
      wrap>
      <v-flex xs12>
        <p>Nothing to see here.</p>
      </v-flex>
    </v-layout>
  </v-container>
</template>

<script>
export default {
  name: "Progress"
};
</script>

<style scoped>
</style>
