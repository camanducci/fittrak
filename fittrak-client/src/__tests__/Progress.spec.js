import { shallowMount } from "@vue/test-utils";

import Progress from "@/pages/Progress";

describe("Progress", () => {
  test("Progress page renders", () => {
    const app = shallowMount(Progress);
    expect(app.html()).toMatchSnapshot();
  });
});
