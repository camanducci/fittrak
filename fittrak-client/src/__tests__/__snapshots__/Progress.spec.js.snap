// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress Progress page renders 1`] = `
<vcontainer-stub tag="div">
  <vlayout-stub tag="div" row="" wrap="">
    <vflex-stub tag="div" xs12="">
      <p>Nothing to see here.</p>
    </vflex-stub>
  </vlayout-stub>
</vcontainer-stub>
`;
